ui/app/AppLayouts/Shell/delegates/ShellGridChatItem.qml

```diff
@@ -42,5 +44,16 @@ ShellGridItem {
         }
     }
 
-    // TODO bottomRowComponent -> last message in this chat
+    bottomRowComponentFillsWidth: true
```

Comment:
You can use sth like:
```qml
width: root.availableWidth
```
on the `bottomRowComponent`; see e.g. the ShellGridWalletItem